tests/unit/serialization/custom_global_serialization_test.py

```diff
@@ -128,3 +139,21 @@ def test_double_register_custom_serializer(self):
 
         with self.assertRaises(ValueError):
             service._registry.safe_register_serializer(TheOtherCustomSerializer, CustomClass)
+
```

Comment:
unnecessary newline. also, there are some unnecessary new lines in the test method below (lines 145 and 147)

Reply:
Okay, fixed them.